docs/extras/modules/data_connection/vectorstores/integrations/meilisearch.ipynb

```diff
@@ -68,87 +91,60 @@
     "import os\n",
     "import getpass\n",
     "\n",
+    "# Prompt for Meilisearch credentials\n",
+    "os.environ[\"MEILI_HTTP_ADDR\"] = getpass.getpass(\"Meilisearch HTTP address and port:\")\n",
+    "os.environ[\"MEILI_MASTER_KEY\"] = getpass.getpass(\"Meilisearch API Key:\")\n",
+    "\n",
+    "# Prompt for other third-party credentials\n",
     "os.environ[\"OPENAI_API_KEY\"] = getpass.getpass(\"OpenAI API Key:\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you don't want to have to send your client or Meilisearch instance address, you can directly configure the [environment variables](https://www.meilisearch.com/docs/learn/configuration/instance_options#environment-variables)."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "os.environ[\"MEILI_HTTP_ADDR\"] = getpass.getpass(\"Meilisearch HTTP address and port:\")"
+    "### Adding text and embeddings\n",
+    "\n",
+    "This example adds text to the Meilisearch vector store along with the associated embeddings."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": [
-    "os.environ[\"MEILI_MASTER_KEY\"] = getpass.getpass(\"Meilisearch API Key:\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Add texts with user-friendly interface"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "from langchain.vectorstores import Meilisearch\n",
     "from langchain.embeddings.openai import OpenAIEmbeddings\n",
     "from langchain.text_splitter import CharacterTextSplitter\n",
     "\n",
-    "embeddings = OpenAIEmbeddings()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 6,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "# OpenAI API key is loaded from environment\n",
```

Comment:
```suggestion
```
This creates a wrapper around OpenAI Embeddings API which will able to create an embedding vector representing the input text.
I don't know if just specifying that the key is loaded from the environment won't create confusion?

Reply:
What would you suggest instead? I'm not sure to grasp the full scope

Reply:
"Generates embeddings for a given text using the OpenAI API thanks to the key loaded via the environment" but it's also explained directly in the Langchain documentation [here](https://js.langchain.com/docs/modules/data_connection/text_embedding/integrations/openai)
So we can just said:
"Will use OpenAI API key loaded from environment\n",